### PR TITLE
Update examples metadata

### PR DESCRIPTION
**Pull request by Richiio**
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [ ] issues: fixes #xxxx
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

**Comment by bryevdv** [comment]
Hi @Richiio Thank you for your PR. The necessary task is to update the existing examples where they are located, but in this PR the new versions of the files are saved as entirely new files at the top level of the repo. Please update your local branch to update (overwrite) the existing files and then push those  changes to update this PR. For reference, it looks like those files are located in these places:

```
examples/plotting/dynamic_color_mapping.py
examples/plotting/theme_glyphs.py
```
Let me know if there are other questions I can address.

**Comment by bryevdv** [comment]
Also, the linter is reporting some issues with the files as they are currently, that will need to be fixed up before merge:
```
E         File contains trailing whitespace: dynamic_color_mapping.py, line 7.
E         File contains trailing whitespace: theme_glyphs.py, line 3.
E         File contains trailing whitespace: theme_glyphs.py, line 8.
```
and
```
E         theme_glyphs.py:5:166: E501 Line too long (235 > 165 characters)
```
For the second, you can remove some of the references (just having a few of the  most relevant refs is fine) or you can add a comment `# noqa` to the end to tell the linter to ignore that line. Unfortunately the line can't be split, the sphinx extension does not understand that.

**Comment by Richiio** [comment]
Hi @bryevdv, would work on this before the end of day and make a new pull request. Thanks for the feedback!

**Comment by Richiio** [comment]
Hi @bryevdv , it's been done.

File: examples/plotting/dynamic_color_mapping.py
Comment (bryevdv) [inline examples/plotting/dynamic_color_mapping.py]:
```suggestion
array as low, high or median.

.. bokeh-example-metadata::
```

File: examples/plotting/dynamic_color_mapping.py
Comment (bryevdv) [inline examples/plotting/dynamic_color_mapping.py:7]:
```suggestion
    :keywords: plot, tooltips, tools
    
```

File: examples/plotting/theme_glyphs.py
Comment (bryevdv) [inline examples/plotting/theme_glyphs.py]:
```suggestion
''' A graph showing how to plot a glyph using your own custom theme.

.. bokeh-example-metadata::
```

File: examples/plotting/theme_glyphs.py
Comment (bryevdv) [inline examples/plotting/theme_glyphs.py]:
```suggestion
    :keywords: glyph, theme
    
'''
```

File: examples/plotting/dynamic_color_mapping.py
Comment (bryevdv) [inline examples/plotting/dynamic_color_mapping.py]: I don't follow the reference to medians, the linear color mapper does not use any median computations. Maybe just:
```suggestion
array.
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks @Richiio just a few suggestions, mostly to keep style consistency across docstrings

**Comment by bryevdv** [comment]
> Hi @bryevdv, would work on this before the end of day and make a new pull request. Thanks for the feedback!

Hi @Richiio its not necessary to make a *new* PR, you can can add and commit changes to your existing local branch, and then `git push` and this existing PR will be updated (this is preferable to keep comments and discussion in one place) Let me know if there are any questions about this

File: examples/plotting/dynamic_color_mapping.py
Comment (Richiio) [reply examples/plotting/dynamic_color_mapping.py]: Hi @bryevdv, I think it does although I may have not explained correctly.

For instance, we have an array for numbers between 1 - 100 and colours red, blue and green. Numbers within the range 1-40 (low) will be given red, numbers within the range 41 - 60 (middle) can be given blue and numbers within the range 61-100 will be given green(high)

So using the color mapper, we have been able to show, maybe with a scatterplot the distribution of the numbers when plotted. When we see a red dot, we know the value there should be between 1-40 without having to hover over it.

Waiting for your feedback on this. Thanks for the rest, would make the correct edits and push it.

**Comment by Richiio** [comment]
Thanks for this!

File: examples/plotting/dynamic_color_mapping.py
Comment (bryevdv) [reply examples/plotting/dynamic_color_mapping.py]: Hi @Richiio when I see "median", I only think of the mathematical definition: 

>noun
Arithmetic, Statistics. the middle number in a given sequence of numbers, taken as the average of the two middle numbers when the sequence has an even number of numbers:
4 is the median of 1, 3, 4, 8, 9.

But this usage in this example has nothing to do with that, so  do think there is room for confusion. I would still request applying the suggestion. (I think "use colours to map range of value" is sufficient to express the main point)

**Comment by Richiio** [comment]
Hi @bryevdv, thanks once again for the feedback! All updated now

**Comment by Richiio** [comment]
The colors below were too dull, so I updated it with brighter colors. It also aids in clearly depicting the variations of the two scatterplots
before1
Plot with updated color
after1

**Comment by bryevdv** [comment]
Hi @Richiio Thanks for the PR. I have to be honest and state that I'm not a fan of pure primary colors in opposition, or just looks very garish to my eye. Please try out and display here a few different combinations from some of the existing categorical palettes in the docs:

https://docs.bokeh.org/en/latest/docs/reference/palettes.html

**Comment by Richiio** [comment]
Hi @bryevdv how about this contrast?
after1

**Comment by Richiio** [comment]
Hi @bryevdv how about this contrast?
after1

**Comment by bryevdv** [comment]
Hi @Richiio Thanks for you patience, I've been sick most of the last few weeks. I think those colors look much better! If we want to update the colors, we will need to generate new thumbnails. Several of the recent contribution period PRs went through that process that you can refer to, but please let me know if I can provide more guidance!

**Comment by bryevdv** [comment]
Also note there are some linter errors (trailing spaces are not allowed):
```
E         File contains trailing whitespace: examples/plotting/dynamic_color_mapping.py, line 8.
E         File contains trailing whitespace: examples/plotting/theme_glyphs.py, line 7.
```

**Comment by bryevdv** [comment]
closing for lack of response to review @Richiio please feel free to open a new PR in the future if you are able to return to this.

**Pull request closed without merging**
